logging/logging.go

```diff
@@ -84,79 +91,41 @@ func SetLogLevel(level int) {
 	}
 }
 
-// GetLogLevel gets logrus log level and returns int Acra log level
-func GetLogLevel() int {
-	if log.GetLevel() == log.DebugLevel {
-		return LogDebug
-	}
-	if log.GetLevel() == log.InfoLevel {
-		return LogVerbose
+// CreateFormatter creates formatter object
+func CreateFormatter(format string) FormatterWrapper {
+	var formatter FormatterWrapper
+	switch strings.ToLower(format) {
+	case JsonFormatString:
+		formatter = JSONFormatter()
+	case CefFormatString:
+		formatter = CEFFormatter()
+	default:
+		formatter = TextFormatter()
```

Comment:
I see that it's the behavior we had before, but is it okay to let the users say `--log_format "cat videos"` and get the default one without any error?

Reply:
Yes, probably it is better to throw some error like `unexpected logs format has been specified`